### PR TITLE
Add dynamic app views

### PR DESCRIPTION
**Pull request by Beardless-sheik**
In this pull request, I: 
- used preprocessed HTML file with embedded Ruby code to show different pages of the app according to the wireframes given
- used layouts and templates for shared content.
- used model & controller methods to dynamically display data

### CODE REVIEW

File: app/views/main/index.html.erb [hunk 1]
```diff
@@ -6,75 +6,25 @@
...
+                <div class="username-text-container border text-center align-middle m-1">
+                  <p class="user-name-text"> <%= user.Name %> </p>
+                  <p class="posts-counter-text"> Number of Posts: <%= user.Posts_Counter %>  </p>
+                </div>
+              </a>
+            </div>
+          </section>
+        <% end %>
+      <% end %>
     </main>
```
Comment (VICTOR-CHUKWU) [inline app/views/main/index.html.erb:28]: - [ ] [Optional] You should do well to put this code in your `users/index.html.erb` file for clarity

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Beardless-sheik , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/main/index.html.erb (hunk 1 above)
Comment (Beardless-sheik) [reply app/views/main/index.html.erb:28]: Thanks for the suggestion

**Review by arslanbisharat — COMMENTED** [review]
Hi @Beardless-sheik, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has the wrong gitflow and already approved. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**